src/main/java/apoc/meta/Meta.java

```diff
@@ -234,12 +237,18 @@ public void rel(int typeId, String typeName, long count) {
             }
 
             public void rel(int typeId, String typeName, int labelId, String labelName, long out, long in) {
-                if (out > 0) relStats.put("(:" + labelName + ")-[:" + typeName + "]->()", out);
-                if (in > 0) relStats.put("()-[:" + typeName + "]->(:" + labelName + ")", in);
+                if (out > 0) {
+                    relStats.put("(:" + labelName + ")-[:" + typeName + "]->()", out);
+                    relStatsCount.put(typeName, out);
```

Comment:
Yes it is global, we add it because the global rel count was always 0. Now now we can get total relationship count for each relType `metaStats.relTypesCount.getOrDefault(entityName, 0L)`